### ISSUE TITLE
Upload process

### ISSUE THREAD

**Issue by Grizzly127** [Kompakkt/Repo#25]
I worked on the upload process:
<img width="925" alt="Bildschirmfoto 2023-08-01 um 11 11 19" src="https://github.com/Kompakkt/Repo/assets/44752496/c759139e-e941-4f98-b220-e0aad946da67">
<img width="902" alt="Bildschirmfoto 2023-08-01 um 11 11 28" src="https://github.com/Kompakkt/Repo/assets/44752496/f9623baa-3f80-4a71-be31-b7f6c113753d">

I think the idea from semantic Kompakkt to keep the options on the right is good, so I used it here.
I divided the options on the right in mandatory and optional settings. 
I corrected the fonts (using Open Sans everywhere)
I adjusted the buttons
I used the new stepper.
I reduced the options in "general information", before there were a lot more options in Kompakkt: 
- [ ] Do we need any of these? 
<img width="375" alt="Bildschirmfoto 2023-08-01 um 11 15 42" src="https://github.com/Kompakkt/Repo/assets/44752496/adba12fd-9c6e-474b-8902-f1b9da355a51">

I had the idea to also use a stepper for the metadata settings, but as it also is a step in the upload process it won't work like that. I just want to keep the idea here: 
<img width="687" alt="Bildschirmfoto 2023-08-01 um 11 19 07" src="https://github.com/Kompakkt/Repo/assets/44752496/b91742cd-95b1-4ad2-a582-9ed307c2d529">

@HeyItsBATMAN @ZetOE @lozanaross

**Comment by ZetOE** [comment]
Hey @HeyItsBATMAN, nice suggestion! I think most of the fields under general information were not used, so they can go right out. I would keep the set of metadata as small as possible as long as nothing important is missing. And the essential stuff for mappings is mandatory anyway.
I agree with the stepper suggestion - looks really good. Maybe @Grizzly127 has another idea of how to combine 2 steppers.

**Comment by HeyItsBATMAN** [comment]
@ZetOE if we want to reduce/simplify the metadata form anyways, maybe we should have a meeting to decide what to keep & what to simplify.
That way we could, at the same time, document how our metadata looks and how it can be mapped to other formats. What do you think?

**Comment by ZetOE** [comment]
Yes, very good suggestion @HeyItsBATMAN . Maybe we can do the same with our annotation data model (but does not necessarily have to happen in the same meeting).

**Comment by ZetOE** [comment]
And: we should include CIDOC mapping.

**Comment by HeyItsBATMAN** [comment]
@Grizzly127 one thing I just noticed: Description should be a larger text area instead of a single line

**Comment by lozanaross** [comment]
@HeyItsBATMAN & @ZetOE I discussed some of the new design proposals with @Grizzly127 and we think this one, alongside https://github.com/Kompakkt/Repo/issues/28 are a bit more complex and should be put on hold, until the higher priority ones - like fixing the nav / action bar, the login dialog, etc are completed. We should perhaps think about a labeling schema that can specify when something is higher priority than other issues. And maybe it's time we started using 'Projects' within the Git org page, to keep all issues across all repos a bit more organized at a glance.

**Comment by Grizzly127** [comment]
Yes, I am closing this issue. For new topics regarding the upload process a new issue will be opened
